package.json

```
@@ -34,18 +34,35 @@
 		}
 	],
 	"type": "commonjs",
-	"main": "PJV.js",
+	"exports": {
+		".": {
+			"import": {
+				"type": "./lib/index.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"type": "./lib/index.d.ts",
+				"default": "./lib/index.js"
+			}
+		},
+		"./package.json": "./package.json"
+	},
+	"main": "lib/index.js",
+	"module": "lib/index.mjs",
+	"types": "lib/index.d.ts",
 	"bin": {
-		"pjv": "./bin/pjv"
+		"pjv": "./lib/bin/pjv.mjs"
 	},
 	"files": [
-		"bin/"
+		"lib/"
 	],
 	"scripts": {
+		"build": "tsup",
 		"format": "prettier .",
 		"lint": "eslint . --max-warnings 0",
 		"prepare": "husky",
-		"test": "vitest"
+		"test": "vitest",
+		"type-check": "tsc --noEmit"
```

Since this is already set in `tsconfig.json`:

```suggestion
		"type-check": "tsc"
```